src/validate-json-schema.js

```diff
@@ -9,6 +10,7 @@ const Ajv = require('ajv');
  */
 function validateJson(json) {
     const ajv = new Ajv({strictTuples: false});
+    addFormats(ajv);
```

Comment:
Do we provide what exactly is not valid for client?

Reply:
@naydav , yes, `validateJson(json)` is invoked during [saving](https://github.com/adobe/aio-template-submission/blob/9aca6c849b7034833ecfe83b06c7fbaff03ee237/src/registry.js#L16-L17) the Template Registry data. It generates [an exception](https://github.com/adobe/aio-template-submission/blob/9aca6c849b7034833ecfe83b06c7fbaff03ee237/src/validate-json-schema.js#L16) for the incorrect data, that is later [caught](https://github.com/adobe/aio-template-submission/blob/9aca6c849b7034833ecfe83b06c7fbaff03ee237/src/add-to-registry.js#L54) and [added](https://github.com/adobe/aio-template-submission/blob/9aca6c849b7034833ecfe83b06c7fbaff03ee237/.github/workflows/add-template.yml#L92) as a comment to the corresponding Github issue.